armi/operators/tests/test_operators.py

```diff
@@ -76,5 +78,105 @@ def test_addInterfaceSubclassCollision(self):
         self.assertEqual(o.getInterface("Third"), interfaceC)
 
 
+class CyclesSettingsTests(unittest.TestCase):
+    """
+    Check that we can correctly access the various cycle settings from the operator.
+    """
+
+    detailedCyclesSettings = """
+metadata:
+  version: uncontrolled
+settings:
+  power: 1000000000.0
+  nCycles: 3
+  cycles:
+    - name: dog
+      cumulative days: [1, 2, 3]
+      power fractions: [0.1, 0.2, 0.3]
+      availability factor: 0.1
+    - cycle length: 10
+      burn steps: 5
+      power fractions: [0.2, 0.2, 0.2, 0.2, 0]
+      availability factor: 0.5
+    - name: ferret
+      step days: [3, R4]
+      power fractions: [0.3, R4]
+  runType: Standard
+"""
+
+    powerFractionsSolution = [
+        [0.1, 0.2, 0.3],
+        [0.2, 0.2, 0.2, 0.2, 0],
```

Comment:
recommend `0.0` for the last one so they're all floats

Reply:
Well I actually wanted to test having an int in there, just to make sure that it actually works, since I anticipate users might very likely put an int in their own inputs.

Was there some other reason why you make this recommendation that maybe I'm overlooking?